index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+<body>
+
+<section class="header">
+    
+    <div class="Top-Nav">
```

Comment:
- [ ] You should be using a `header` tag here instead of a `div` as specified in the project requirements. Also, please note that this element should be outside this main element.